pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -1617,12 +1618,34 @@ func (gce *GCECloud) GetZone() (cloudprovider.Zone, error) {
 	}, nil
 }
 
+// GCE does not have tags in Disk objects. We store the tags in Description
+// field, encoded in JSON. encodeDiskTags prepares such JSON.
+func (gce *GCECloud) encodeDiskTags(tags map[string]string) (string, error) {
```

Comment:
I don't know if unexported functions are part of Godoc, but following the same naming convention can't hurt.